src/components/create-password/create-password-controller.ts

```diff
@@ -11,6 +11,12 @@ import {
 } from "../../utils/validation";
 
 export function createPasswordGet(req: Request, res: Response): void {
+  if (
+    req.session.user.featureFlags &&
+    req.session.user.featureFlags.updatePasswordHintTextVersion === "2"
+  ) {
+    return res.render("create-password/ab-tests/index-variant.njk");
```

Comment:
Definitely a question NOT a suggestion, but do we actually want to display the variant when the user hasn't accepted cookies or does the fact non-cookie-consent users can't be tracked via GA mean that there's no learning benefit for us in showing them the variant, and it's slightly 'safer' for their user experience to show them the 'tried and tested' option.